agentao/helpers/helpers.py

```diff
@@ -4,11 +4,11 @@
 import math
 from git import Repo
 
-from agentao.helpers.clients import LOGGER
 from agentao.helpers.constants import PRICING_DATA_PER_MILLION_TOKENS
 
+from logging import Logger
 
-def clone_repo(author_name: str, repo_name: str, base_path: Path) -> Path:
+def clone_repo(author_name: str, repo_name: str, base_path: Path, logger: Logger) -> Path:
```

Comment:
We should do this so that we can test helpers individually without having to make logger params every time. I added suggestions doing this, you should be able to just accept them.
```suggestion
def clone_repo(author_name: str, repo_name: str, base_path: Path, logger: Logger = None) -> Path:
    logger = logger or logging.getLogger(__name__)
```

Reply:
imo we shouldnt do this as itll mean we wont see an error if loggers arent passed to child functions (meaning we wont have context on the log)